assets/src/editor/events/hocs/with-editor-ticket-entities.js

```diff
@@ -0,0 +1,12 @@
+import { withSelect } from '@wordpress/data';
+import { createHigherOrderComponent } from '@wordpress/compose';
+
+export default createHigherOrderComponent(
+	withSelect( ( select ) => {
+		return {
+			ticketEntities: select( 'eventespresso/core' )
+				.getTickets(),
+		};
+	} ),
+	'withEditorTicketEntities'
```

Comment:
Think of this hoc replacing the need to pass down the `allTicketEntities` prop everywhere.